catalog/dags/providers/provider_api_scripts/europeana.py

```diff
@@ -71,6 +71,7 @@ def get_record_data(self, data: dict) -> dict | None:
                 "license_info": self._get_license_info(data),
                 "filetype": self._get_filetype(item_data),
                 "filesize": self._get_filesize(item_data),
+                "thumbnail_url": self._get_thumbnail(item_data),
```

Comment:
The problem is passing `item_data` here, which doesn't contain the thumbnail, instead of `data`. Replacing it made the DAG collect the thumbs for me.

```suggestion
                "thumbnail_url": self._get_thumbnail(data),
```

Reply:
Thanks for pointing this out @krysal. It's funny how I wrote the "running DAGs locally" guide but still don't fully understand where to check for what, reason I couldn't verify this. At this point we might just need a video demo 😅